src/test_meta_srv.c

```diff
@@ -24,31 +24,30 @@
 
 /**
  * Performs the META test.
- * @param ctlsockfd Client control socket descriptor
+ * @param ctl Client control Connection
  * @param agent UNUSED Web100 agent used to track the connection
  * @param testOptions The test options
  * @param conn_options The connection options
  * @return 0 - success,
- *          >0 - error code.
- *          Error codes:
- *           -1 - Cannot write message header information while attempting to send
- *           		 TEST_START message
- *          -2 - Cannot write message data while attempting to send
- *           		 TEST_START message
- *          1 - Message reception errors/inconsistencies
- *			2 - Unexpected message type received/no message received due to timeout
- *			3 - Received message is invalid
- *			4 - Invalid data format in received message
+ *         Error codes:
+ *          -2 - Cannot write message data while attempting to send TEST_START
+ *          -1 - Cannot write message header information while attempting to
+ *               send TEST_START message
+ *           1 - Message reception errors/inconsistencies
+ *           2 - Unexpected message type received/no message received due to
+ *               timeout
+ *           3 - Received message is invalid
+ *           4 - Invalid data format in received message
  */
 
-int test_meta_srv(int ctlsockfd, tcp_stat_agent* agent,
-                  TestOptions* testOptions, int conn_options) {
+int test_meta_srv(Connection *ctl, tcp_stat_agent *agent,
```

Comment:
75% is one way, 25% the other.  * on the variable name is the right way of doing things, though.  So as I see 'em, I'm fixing 'em.